devpack/build.gradle

```diff
@@ -2,4 +2,8 @@ description 'neow3j: Java Development Package for Neo Smart Contracts'
 
 dependencies {
     compile project(':contract')
-}
\ No newline at end of file
+}
+
+compileJava {
+    options.debugOptions.debugLevel = "source,lines,vars"
```

Comment:
so, this is necessary only for the `devpack` module?

Is this necessary to set in the user's `build.gradle` project as well? (if users wants debug info)